src/main/java/org/gridsuite/explore/server/services/ExploreService.java

```diff
@@ -285,4 +257,22 @@ public void createNetworkModifications(List<ElementAttributes> modificationAttri
             }
         });
     }
+
+    public void duplicateContingencyList(UUID contingencyListsId, UUID targetDirectoryId, String userId, ContingencyListType contingencyListType) {
```

Comment:
move code near `createScriptContingencyList` for readability

Reply:
done